doc/_static/arch/system_containers_components.svg

```diff
@@ -0,0 +1,1629 @@
+link default_qubit to other--><!--MD5=[5765bccd7314636ac8a75b927a991b7b]
+link python to other--><!--MD5=[5765bccd7314636ac8a75b927a991b7b]
+link python to other--><path d="M484,134.6999 C484,172.5999 484,227.5499 484,267.4999 " fill="none" id="python-to-other" style="stroke:#666666;stroke-width:1.0;"/><polygon fill="#666666" points="484,275.5399,487,267.5399,481,267.5399,484,275.5399" style="stroke:#666666;stroke-width:1.0;"/><text fill="#666666" font-family="sans-serif" font-size="12" font-weight="bold" lengthAdjust="spacing" textLength="30" x="485" y="193.7479">Uses</text><!--MD5=[fe122ec3631594ac4371c7e119759707]
+link developer to python--><path d="M304.51,90.0999 C343.7,90.0999 389.78,90.0999 425.44,90.0999 " fill="none" id="developer-to-python" style="stroke:#666666;stroke-width:1.0;"/><polygon fill="#666666" points="433.8,90.0999,425.8,87.0999,425.8,93.0999,433.8,90.0999" style="stroke:#666666;stroke-width:1.0;"/><text fill="#666666" font-family="sans-serif" font-size="12" font-weight="bold" lengthAdjust="spacing" textLength="30" x="354.25" y="67.2479">Uses</text><text fill="#666666" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="93" x="322.75" y="83.7001">[Win/MacOS/Lnx]</text><!--MD5=[db00fa398594d0e602e6a4e604277272]
+link default_qubit to lightning_qubit--><path d="M151,363.7399 C151,403.9899 151,464.6899 151,511.5599 " fill="none" id="default_qubit-to-lightning_qubit" style="stroke:#666666;stroke-width:1.0;"/><polygon fill="#666666" points="151,519.8499,154,511.8499,148,511.8499,151,519.8499" style="stroke:#666666;stroke-width:1.0;"/><text fill="#666666" font-family="sans-serif" font-size="12" font-weight="bold" lengthAdjust="spacing" textLength="118" x="158.5" y="406.2479">Ops and data pointer</text><text fill="#666666" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="91" x="170.5" y="422.7001">[Provides default</text><text fill="#666666" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="128" x="155" y="438.9002">Python plugin methods]</text><!--MD5=[e74efe3d600ba974dbb7e83863c41cd6]
```

Comment:
What do we mean by "Ops and data pointer"?

Reply:
Good point --- "Operations and pointer to data". I should be more explicit on this